apiml-static-reg.yaml.template

```diff
@@ -12,7 +12,7 @@ services:
     apiInfo:
       - apiId: org.zowe.zss
         gatewayUrl: api/v1
-        # swaggerUrl: TODO
+        swaggerUrl: https://${ZOWE_EXPLORER_HOST}:${ZOWE_ZLUX_SERVER_HTTPS_PORT}/api-docs/zss
```

Comment:
this would probably fail if zlux was not running, which is possible. could we use some env var to make it undefined in that case?

Reply:
Sure, we can make the whole url undefined if thats the case. Np

Reply:
@1000TurquoisePogs if swagger is unreachable it wont show up, we should not corrupt the file though i believe

Reply:
made 8544 as default